server/config.sample.json

```diff
@@ -3,7 +3,7 @@
   "RPC_PORT": 8545,
 
   "SECRET": "jpLHCCvYF0FNXkpXKfLmS7XQcOEzFhzH0f9kL1YV8YFSdIXPAArWWrRtpliILqsH",
-  "BITLY_TOKEN": <token>,
-  "FUM_TOKEN": <token>,
-  "FUM_BASEURL": <url>
+  "BITLY_TOKEN": "dummytoken",
```

Comment:
I'm not sure if having these as valid strings is more helpful than not, since I guess the original idea was that if the JSON is not parseable it forces someone to look at it first...